### ISSUE TITLE
Deluge app "Illegal string offset 'name'"

### ISSUE THREAD

**Issue by made2140** [linuxserver/Heimdall#464]
Trying to edit Deluge link, and this happens:
docker image: **latest**

```
ErrorException thrown with message "Illegal string offset 'name' (View: /config/www/SupportedApps/Deluge/config.blade.php) (View: /config/www/SupportedApps/Deluge/config.blade.php) (View: /config/www/SupportedApps/Deluge/config.blade.php)"

Stacktrace:
#77 ErrorException in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:290
#76 ErrorException in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:290
#75 ErrorException in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:290
#74 ErrorException in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:290
#73 Illuminate\Foundation\Bootstrap\HandleExceptions:handleError in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:290
#72 Collective\Html\FormBuilder:input in /var/www/localhost/heimdall/vendor/laravelcollective/html/src/FormBuilder.php:336
#71 Collective\Html\FormBuilder:password in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:237
#70 Illuminate\Support\Facades\Facade:__callStatic in /var/www/localhost/heimdall/storage/framework/views/fb278cc0897220f39be0c551654f29571bafa9d5.php:10
#69 include in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php:43
#68 Illuminate\View\Engines\PhpEngine:evaluatePath in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php:59
#67 Illuminate\View\Engines\CompilerEngine:get in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:142
#66 Illuminate\View\View:getContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:125
#65 Illuminate\View\View:renderContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:90
#64 Illuminate\View\View:render in /var/www/localhost/heimdall/storage/framework/views/0400c13837702c90caf43da85ae3675fbff580e3.php:95
#63 include in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php:43
#62 Illuminate\View\Engines\PhpEngine:evaluatePath in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php:59
#61 Illuminate\View\Engines\CompilerEngine:get in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:142
#60 Illuminate\View\View:getContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:125
#59 Illuminate\View\View:renderContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:90
#58 Illuminate\View\View:render in /var/www/localhost/heimdall/storage/framework/views/8d91c3cd0f3a13839dbe5eac67e46f31e58dc512.php:5
#57 include in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php:43
#56 Illuminate\View\Engines\PhpEngine:evaluatePath in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php:59
#55 Illuminate\View\Engines\CompilerEngine:get in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:142
#54 Illuminate\View\View:getContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:125
#53 Illuminate\View\View:renderContents in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/View.php:90
#52 Illuminate\View\View:render in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Http/Response.php:42
#51 Illuminate\Http\Response:setContent in /var/www/localhost/heimdall/vendor/symfony/http-foundation/Response.php:202
#50 Symfony\Component\HttpFoundation\Response:__construct in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:750
#49 Illuminate\Routing\Router:toResponse in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:722
#48 Illuminate\Routing\Router:prepareResponse in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:682
#47 Illuminate\Routing\Router:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:30
#46 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/app/Http/Middleware/CheckAllowed.php:42
#45 App\Http\Middleware\CheckAllowed:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#44 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#43 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php:41
#42 Illuminate\Routing\Middleware\SubstituteBindings:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#41 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#40 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:75
#39 Illuminate\Foundation\Http\Middleware\VerifyCsrfToken:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#38 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#37 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php:49
#36 Illuminate\View\Middleware\ShareErrorsFromSession:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#35 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#34 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:63
#33 Illuminate\Session\Middleware\StartSession:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#32 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#31 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php:37
#30 Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#29 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#28 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php:66
#27 Illuminate\Cookie\Middleware\EncryptCookies:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#26 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#25 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:104
#24 Illuminate\Pipeline\Pipeline:then in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:684
#23 Illuminate\Routing\Router:runRouteWithinStack in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:659
#22 Illuminate\Routing\Router:runRoute in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:625
#21 Illuminate\Routing\Router:dispatchToRoute in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:614
#20 Illuminate\Routing\Router:dispatch in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:176
#19 Illuminate\Foundation\Http\Kernel:Illuminate\Foundation\Http\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:30
#18 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/fideloper/proxy/src/TrustProxies.php:57
#17 Fideloper\Proxy\TrustProxies:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#16 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#15 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:31
#14 Illuminate\Foundation\Http\Middleware\TransformsRequest:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#13 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#12 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:31
#11 Illuminate\Foundation\Http\Middleware\TransformsRequest:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#10 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#9 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php:27
#8 Illuminate\Foundation\Http\Middleware\ValidatePostSize:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#7 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#6 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php:62
#5 Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:163
#4 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php:53
#3 Illuminate\Routing\Pipeline:Illuminate\Routing\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:104
#2 Illuminate\Pipeline\Pipeline:then in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:151
#1 Illuminate\Foundation\Http\Kernel:sendRequestThroughRouter in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:116
#0 Illuminate\Foundation\Http\Kernel:handle in /var/www/localhost/heimdall/public/index.php:55
```

**Comment by unintended-consequences** [comment]
Same as #433.

[https://github.com/linuxserver/Heimdall/issues/433](url)

**Comment by malikaygar** [comment]
Has anyone found a fix to this? Thread #433 hasn't resolved this yet.
